templates/internal/appName/config.go.tpl

```diff
@@ -10,13 +10,12 @@ package {{ stencil.ApplyTemplate "goPackageSafeName" }} //nolint:revive // Why:
 
 import (
 	"context"
-	"os"
-	"strings"
 
 	"github.com/getoutreach/gobox/pkg/cfg"
 	"github.com/getoutreach/gobox/pkg/log"
-	"github.com/getoutreach/gobox/pkg/events"
+	{{- if has "kafka" (stencil.Arg "serviceActivities") }}
 	"github.com/getoutreach/services/pkg/find"
```

Comment:
This isn't an OSS repo btw (not ur fault but I see it so I'll call it out)

Reply:
yeah ugh we'll have to add a hook here.  i'll open a ticket.  that'll be more invasive surgery.